src/app/modules/card-cluster/card-cluster.component.html

```diff
@@ -52,18 +52,20 @@
     class="ds-l-md-col--12 ds-l-sm-col--12"
     id="followCard"
   >
-    <div class="ds-l-row">
+    <div class="ds-l-row" [attr.role]="cardArray.showRadioButton ? 'radiogroup' : undefined">
       <ng-container *ngFor="let card of cardArray.cluster; let x = index">
         <div class="ds-l-col" [ngClass]="(x + 1) % 2 === 0 ? 'padding-L-0' : ''">
           <div
-            id="{{ 'CardCluster' + (x + 1) }}"
-            [ngClass]="{ active: cardArray.buttonClicked === 'CardCluster' + (x + 1), disabled: card.disabled }"
+            id="{{ card.id }}"
+            [ngClass]="{ active: cardArray.buttonClicked === card.id, disabled: card.disabled }"
             class="{{ 'pointer card cardHeight ds-u-padding--1 ds-u-margin-bottom--2 ' + card.classButton }}"
             (click)="passAction($event)"
             (keyup.enter)="passAction($event)"
-            role="button"
+            [attr.role]="cardArray.showRadioButton ? 'radio' : 'button'"
             [attr.tabindex]="card.disabled ? null : 0"
             [attr.aria-label]="card.ariaLabel || card.name"
+            [attr.aria-describedby]="card.ariaDecribedbyName ? card.id + 'name' : undefined"
```

Comment:
```suggestion
            [attr.aria-describedby]="card.ariaDescribedbyName ? card.id + 'name' : undefined"
```

Reply:
Fixed typo and camelCase. Added hyphen `'name'` -> `'-name'`